stake-pool/js/src/index.ts

```diff
@@ -352,6 +378,11 @@ export async function withdrawStake(
     stakePoolAddress,
   );
 
+  let stakeReceiverAccount = null;
+  if (stakeReceiver) {
+    stakeReceiverAccount = await getStakeAccount(connection, stakeReceiver);
```

Comment:
I think so , with  ```ParsedAccountData type``` i am checking the owner and  ```parsed``` would be there for an uninitialized stake account since it defines the type for a stake account also.

```
export type ParsedAccountData = {
  /** Name of the program that owns this account */
  program: string;
  /** Parsed account data */
  parsed: any;
};
```

Reply:
Could you test just that function with an uninitialized stake account and make sure that it works?